lib/internal/process/per_thread.js

```diff
@@ -6,18 +6,21 @@
 
 const {
   ArrayIsArray,
+  ArrayPrototypeMap,
+  ArrayPrototypePush,
+  ArrayPrototypeSplice,
   BigUint64Array,
   Float64Array,
   NumberMAX_SAFE_INTEGER,
-  ObjectDefineProperties,
   ObjectDefineProperty,
   ObjectFreeze,
-  ObjectGetOwnPropertyDescriptors,
+  ReflectApply,
   RegExpPrototypeTest,
-  Set,
-  SetPrototype,
-  SetPrototypeHas,
+  SafeSet,
+  StringPrototypeEndsWith,
   StringPrototypeReplace,
+  StringPrototypeStartsWith,
+  StringPrototypeSubstr,
```

Comment:
```suggestion
  StringPrototypeSlice,
  StringPrototypeStartsWith,
```

To avoid using a deprecated method